### PR TITLE
Refactor unit & integration tests

### PR DESCRIPTION
**Pull request by ethan605**
# Problem

The current `npm run test:unit` and `npm run test:integration` tasks doesn't look-up and run the correct unit & integration tests in the project

# Solution

Refactor to have:

- All unit tests placed under `__tests__/` folder (following Jest convention)
- All integration tests placed under `__integrations__/` folder

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [n/a] Has the CHANGELOG been updated?
- [n/a] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [n/a] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1320-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1320-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1320-pr-onfido-sdk-ui-onfido.surge.sh

**Review by kopijunkie — APPROVED** [review]
LGTM

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1320-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1320-pr-onfido-sdk-ui-onfido.surge.sh

File: src/Tracker/__mocks__/safeWoopra.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-empty-function, @typescript-eslint/explicit-module-boundary-types */
+// @TODO: Put proper implementation to test Tracker
```
Comment (kopijunkie) [inline src/Tracker/__mocks__/safeWoopra.js:2]: Is there a ticket in the backlog to track this piece of work to do?

**Review by kopijunkie — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -67,7 +69,11 @@
     "transform": {
       "^.+\\.(js|jsx)$": "babel-jest",
       "^.+\\.(ts|tsx)$": "ts-jest"
-    }
+    },
+    "testMatch": [
+      "**/__tests__/**/*.[jt]s?(x)",
+      "**/__integrations__/**/*.integration.[jt]s?(x)"
+    ]
```
Comment (stefaniacardenas) [inline package.json:76]: Great, I was about to raise this issue as I just added a new test and I noticed that it was not being executed.

**Review by stefaniacardenas — APPROVED** [review]

File: src/Tracker/__mocks__/safeWoopra.js (hunk 1 above)
Comment (ethan605) [reply src/Tracker/__mocks__/safeWoopra.js:2]: Nope, with the most basic mock, everything just is a dummy function. I put a `@TODO` here so if anyone wants to test Woopra further, the mocks should go here.

Comment (kopijunkie) [reply src/Tracker/__mocks__/safeWoopra.js:2]: Ah, ok. Understood.

**Pull request merged**